lib/OpenQA/Utils.pm

```diff
@@ -875,6 +875,10 @@ sub read_test_modules {
     my $testresultdir = $job->result_dir();
     return unless $testresultdir;
 
+    my $modules_spent_time_file = path($testresultdir, "modules_spent_time.json");
+    my $modules_spent_time_data;
+    $modules_spent_time_data = decode_json($modules_spent_time_file->slurp) if (-e $modules_spent_time_file);
```

Comment:
This could be one line. And this line might throw an exception. Maybe we just want to ignore broken timing information instead of not rendering the page at all (error 500) so it would make sense to catch the exception. (Render a warning would be nice, though.)